python/eos/data/pypmc.py

```diff
@@ -155,6 +156,49 @@ def create(path, density):
         with open(os.path.join(path, 'description.yaml'), 'w') as description_file:
             yaml.dump(description, description_file, default_flow_style=False)
 
+    @staticmethod
+    def _cartesian_product(densityA, densityB):
+        """ Compute the cartesian product of two mixture densities. The means and covariances
```

Comment:
Is this computed or constructed? Nit picking, but if other changes crop up, worthwhile to address?